Difference between generated adj matrix (#29)
SpecialeBajs/Rapport: article/experiment.tex

\section{Experiment}

\section{Simple price aware extension of LightGCN}
To see the extension of LightGCN, we compare Price-Aware Recommendation, LightGCN and the adapted LightGCN with Price and category nodes in the input data. 

> The hyperparameter $X$ is tested with the values of $0.0$, $0.5$, $1.0$ and $2.0$ to see the effect of this as input data has on the final result.

Reviewer: ```suggestion
The hyperparameter $X$ is experimented with.  
We test it with the values of $0.0$, $0.5$, $1.0$ and $2.0$ to see what effect the input data has on the final result.
```
